### PR TITLE
#4117 New hover card for user profile

### LINKED ISSUE #4117: New hover card
Maybe it would be nice to redesign hovercard to neobrutalism as well.

Probably we can add it to landing page.

Currently would be interesting add to showing art under address created or collected. 

image

image

## Ref
- https://hic.af/

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #4117
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2022-11-12 at 6 56 36 PM
Screen Shot 2022-11-12 at 7 12 38 PM

### CODE REVIEW

File: components/identity/module/IdentityPopoverFooter.vue [hunk 1]
```diff
@@ -1,38 +1,82 @@
...
+      <h6 class="popover-user-heading pb-2">
+        {{ $t('profile.highestSales') }}
+      </h6>
+      <div class="is-flex sold-items">
+        <div v-for="nft in soldItems" :key="nft.id" class="sold-item">
+          <GalleryCard
+            :id="nft.id"
+            hide-name
+            :metadata="nft.metadata"
+            :data-cy="soldItems.indexOf(nft)" />
```
Comment (kkukelka) [inline components/identity/module/IdentityPopoverFooter.vue:39]: is this data-cy used in any test?

**Review by kkukelka — APPROVED** [review]
lgtm but one question

**Comment by yangwao** [comment]
pay 100 usd

great, wfm

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 24.83 USD/KSM ~ 4.027 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x8f2641aa3cf5e5c68600ef4c9b3656666104a95c0990641383f53296869d5fd6](https://kusama.subscan.io/extrinsic/0x8f2641aa3cf5e5c68600ef4c9b3656666104a95c0990641383f53296869d5fd6)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**